tests/restaurant.spec.js

```diff
@@ -51,40 +51,54 @@ const createMenu = require('../src/restaurant');
+    assert.deepStrictEqual(typeof createMenu({ objetoQualquer: 'objetoQualquer' }), 'object');
     // --------------------------------------------------------------------------------------
-    // TESTE 2: Verifique que, dado que a função createMenu foi chamada com o objeto: `{ food: {}, drink: {} }`, verifique que 'objetoRetornado.fetchMenu' retorna um objeto cujas chaves são somente `food` e `drink`.
+    // TESTE 2: Verifique que, dado que a função createMenu foi chamada com o objeto: `{ food: {}, drink: {} }`, verifique que 'objetoRetornado.fetchMenu' 
+    //retorna um objeto cujas chaves são somente `food` e `drink`.
     // ```
     // const objetoRetornado = createMenu(objetoQualquer);
     // objetoRetornado.fetchMenu // Retorno: { food: {}, drink: {}}
     // ```
+    assert.deepStrictEqual(Object.keys(createMenu({ food: {}, drink: {} }).fetchMenu), ['food', 'drink']);
     // Agora faça o TESTE 3 deste arquivo.
     // --------------------------------------------------------------------------------------
     // TESTE 3: Verifique que o menu passado pra função createMenu é identico ao menu recuperado pela função 'objetoRetornado.fetchMenu'
     // ```
     // const objetoRetornado = createMenu(objetoQualquer);
     // objetoRetornado.fetchMenu // Retorno: objetoQualquer
     // ```
+    assert.deepStrictEqual(createMenu({ objetoQualquer: 'objetoQualquer' }).fetchMenu, { objetoQualquer: 'objetoQualquer' });
     // Agora faça o TESTE 4 deste arquivo.
     // --------------------------------------------------------------------------------------
     // TESTE 4: Verifique que 'objetoRetornado.consumption', após a criação do menu, retorna um array vazio.
     // ```
     // const objetoRetornado = createMenu(objetoQualquer);
     // objetoRetornado.consumption // Retorno: []
     // ```
+    assert.deepStrictEqual(createMenu({ food: {}, drink: {} }).consumption, []);
     // Agora faça o PASSO 2 no arquivo `src/restaurant.js`.
     // --------------------------------------------------------------------------------------
-    // TESTE 5: Verifique que chamar uma função associada à chave `order` no objeto retornado, passando uma string como parâmetro, como `objetoRetornado.order('coxinha')`, tal string é adicionada ao array retornado em `objetoRetornado.consumption
+    // TESTE 5: Verifique que chamar uma função associada à chave `order` no objeto retornado, passando uma string como parâmetro, como 
+    //`objetoRetornado.order('coxinha')`, tal string é adicionada ao array retornado em `objetoRetornado.consumption
     // ```;
     // const objetoRetornado = createMenu(objetoQualquer);
     // objetoRetornado.order("coxinha");
     // objetoRetornado.comsuption // Retorno: ["coxinha"]
     // ```
+    const objetoRetornado = createMenu({
+      food: { 'coxinha': 3.90, 'sanduiche': 9.90, 'sashimi': 15.20, 'sopa': 20.50 },
+      drinks: { 'agua': 3.90, 'cerveja': 6.90 }
+    });
+
```

Comment:
Primeiro bom trabalho! 
Uma pequena sugestão seria mudar o **objetoRetornado** de **const** para **let**, com isso você pode alterar os valores da variável objetoRetornado antes de qualquer teste se necessário.
```suggestion
    let objetoRetornado = createMenu({
      food: { 'coxinha': 3.90, 'sanduiche': 9.90, 'sashimi': 15.20, 'sopa': 20.50 },
      drinks: { 'agua': 3.90, 'cerveja': 6.90 }
    });

```

Reply:
Obrigada!! Obrigada pela sugestão, vou realizar a alteração!

Reply:
Boa!